lang/fortran/Makefile

```diff
@@ -0,0 +1,12 @@
+FORTRAN= /usr/bin/gfortran
+
+FFLAGS= -Wall -fPIC -pie -distcc
+
+LDFLAGS = --static-pie -static-libgcc
+all: helloworld
+
+helloworld: helloworld.f90
+	$(FORTRAN) $(FFLAGS) $^ -o $@ $(LDFLAGS)
+	 rm -f helloworld-*
```

Comment:
Is this necessary?
If I remove it, there are no additional files created in the working directory.

If it is necessary, place it under the `clean` rule, not the build rule, if not just remove it.